From 9094c7924d8626a72789cb887548d4d1bc99d892 Mon Sep 17 00:00:00 2001
From: Stefano Stabellini <stefano.stabellini@amd.com>
Date: Sat, 14 Jan 2023 17:56:47 +0200
Subject: [PATCH 11/62] build: do not set have_xen_pci_passthrough for aarch64
 targets

have_xen_pci_passthrough is only used for Xen x86 VMs.

Signed-off-by: Stefano Stabellini <stefano.stabellini@xilinx.com>
[ported to v7.0.0]
Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 meson.build | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index be7e554d26..924e7bea26 100644
--- a/meson.build
+++ b/meson.build
@@ -350,7 +350,11 @@ if get_option('kvm').allowed() and targetos == 'linux'
 endif
 if get_option('xen').allowed() and 'CONFIG_XEN_BACKEND' in config_host
   accelerators += 'CONFIG_XEN'
-  have_xen_pci_passthrough = get_option('xen_pci_passthrough').allowed() and targetos == 'linux'
+  if cpu in ['x86', 'x86_64']
+    have_xen_pci_passthrough = get_option('xen_pci_passthrough').allowed() and targetos == 'linux'
+  else
+    have_xen_pci_passthrough = false
+  endif
 else
   have_xen_pci_passthrough = false
 endif
-- 
2.34.1

